jetty-start/src/main/java/org/eclipse/jetty/start/BaseBuilder.java

```diff
@@ -264,56 +271,65 @@ public boolean accept(Path entry) throws IOException
             StartLog.warn("Use of both %s and %s is deprecated", getBaseHome().toShortForm(startd), getBaseHome().toShortForm(startini));
 
         builder.set(useStartD ? new StartDirBuilder(this) : new StartIniBuilder(this));
-        newlyAdded.stream().map(modules::get).forEach(module ->
-        {
-            String ini = null;
-            try
+        newlyAdded.stream()
```

Comment:
This method is a monster with lots of complex conditionals and loops.
I think perhaps now we probably need to split and/or better document each section of this method.

Reply:
But not for this PR, a new PR for that is a better choice.

Reply:
Then this PR at least needs more comments to explain how/why it fixes the problem